docs/configuration-reference/components/web-ui.md

```diff
@@ -46,25 +47,25 @@ Table of all the arguments accepted by the component.
 Example:
 
 | Argument                             | Description                                                                                                                                   | Default                  | Type   | Required |
-|--------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|--------|----------|
-| `namespace`                          | Namespace where Headlamp will be installed.                                                                                                   | "lokomotive-system"      | string | false    |
-| `ingress`                            | Configuration block for exposing Headlamp through an Ingress resource.                                                                        | -                        | block  | false    |
-| `ingress.host`                       | Used as the `hosts` domain in the Ingress resource for headlamp that is automatically created.                                                | -                        | string | true     |
-| `ingress.class`                      | Ingress class to use for the Headlamp Ingress.                                                                                                | `contour`                | string | false    |
+|--------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|--:-:---------------------|--:-:---|--:-:-----|
+| `namespace`                          | Namespace where the Web UI will be installed.                                                                                                 | "lokomotive-system"      | string | false    |
+| `ingress`                            | Configuration block for exposing the Web UI through an Ingress resource.                                                                      | -                        | block  | false    |
```

Comment:
If `ingress.host` is required. Then I think `ingress` should also be required and set to true.

Reply:
The reasoning I've been using regarding ingress blocks is that `ingress` is not required, but if you include the block then all its arguments are required. That's why `ingress` is marked as optional and the others are required. Does it make sense?

Reply:
Yes